css/index.css

```diff
@@ -73,10 +73,17 @@ h1, h2, h3, h4, h5 {
 /* Container Styles */
 
 .container {
-	width: 880px;
+	max-width: 880px;
 	margin: 0 auto;
 }
 
+@media ( max-width: 800px ) {
+    .container {
+        max-width: 745px;
```

Comment:
You may want to try a responsive unit here, such as %, this will help alleviate the issues that occur at/very close to the breakpoint.